### ISSUE TITLE
AADSTS7000112: Application 'f8119f58-4523-44c1-ab72-b2e0c815bd6a'(Presence Light) is disabled

### ISSUE THREAD

**Issue by fdarrenlacy** [isaacrlevin/presencelight#935]
**Describe the bug**
I can no longer signin to the app with my corp credentials.  It says your AAD Presence Light is disabled

**To Reproduce**
1. Go to Teams Status
2. Click on SIGN IN button
3. Opens browser windows to signin, generates error

Request Id: 25408c00-756b-4260-b880-6eaaaf9f4100
Correlation Id: dd9cbc38-68e5-4f6f-9e4c-e95bcdb7b6fa
Timestamp: 2024-10-31T23:14:19Z
Message: AADSTS7000112: Application 'f8119f58-4523-44c1-ab72-b2e0c815bd6a'(Presence Light) is disabled.

**Expected behavior**
Successful signin as before

**Screenshots**
![2024-10-31_160956](https://github.com/user-attachments/assets/48d10418-3280-410f-af90-0dfae106bdeb)

## Further details
![2024-10-31_162729](https://github.com/user-attachments/assets/cf1dcefb-3d8c-4586-8b4e-f835c47264f8)

- PresenceLight App Type (Desktop)
- PresenceLight Build (Nightly), same result with App store version
- PresenceLight Version (5.8.7t)
- Did you clear settings.json? (Yes)
- ASP.NET Core version

Microsoft.AspNetCore.App 8.0.10 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 6.0.35 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 8.0.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 6.0.35 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 8.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

**Comment by fdarrenlacy** [comment]
I have confirmed this is now closed and working normally.  Would love to know what was done.  I work at MS and thus could see in the Azure portal that the entra app for presence light was disabled.  Today I can see it is once again enabled.  There are no owners, who did this?

![PL](https://github.com/user-attachments/assets/0fb7f1c9-4ee2-462c-8628-bfc864427f97)

After your followup, I am happy to close this ticket.
